testsuite/.rubocop_todo.yml

```diff
@@ -601,6 +603,7 @@ Style/StringLiterals:
     - 'features/step_definitions/datepicker_steps.rb'
     - 'features/step_definitions/salt_steps.rb'
     - 'features/step_definitions/setup_steps.rb'
+    - 'features/step_definitions/vm_steps.rb'
```

Comment:
Can we fix whatever is need to don't include this extra excluded file?

Reply:
Fixed.